src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -0,0 +1,126 @@
+package seedu.address.logic.commands;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.assignment.Assignment;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Github;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.Telegram;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Import command to load data from specified CSV file location.
+ */
+public class ImportCommand extends Command {
```

Comment:
Import fails when there is a `,` in one of the fields. e.g. when there is a `,` in the address field, the telegram fields fails to parse.